app/moved_urls.yml

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
I think we might be missing one for this page: https://deploy-preview-4373--kongdocs.netlify.app/gateway/2.8.x/availability-stages/

This one won't take me to the 3.0 docs.